.kokoro/release/common.cfg

```diff
@@ -0,0 +1,49 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Download trampoline resources.
+gfile_resources: "/bigstore/cloud-devrel-kokoro-resources/trampoline"
+
+# Use the trampoline script to run in docker.
+build_file: "gapic-generator-java/.kokoro/trampoline.sh"
+
+# Configure the docker image for kokoro-trampoline.
+env_vars: {
+  key: "TRAMPOLINE_IMAGE"
+  value: "gcr.io/cloud-devrel-kokoro-resources/java8"
```

Comment:
Unlike the other repos where we can compile with Java 11 and make the source compatible with Java 8, we have to compile gapic-generator-java in Java 8. It's not a dependency to client libraries, it has to load classes from Java 8 instead of Java 11 during compilation time.